charts/karpenter/templates/configmap.yaml

```diff
@@ -10,4 +10,49 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 data:
-  {{- list .Values.settings "" | include "flattenSettings" | indent 2 }}
+{{- with .Values.settings.batchMaxDuration }}
+  batchMaxDuration: "{{ . }}"
+{{- end }}
+{{- with .Values.settings.batchIdleDuration }}
+  batchIdleDuration: "{{ . }}"
+{{- end }}
+{{- with .Values.settings.aws.assumeRoleARN }}
```

Comment:
Looks like feature gates also need to be added here.

Reply:
Fixed. Good catch 👍